packages/react-aria-components/example/index.css

```diff
@@ -3,7 +3,7 @@ html {
   font-family: system-ui;
 }
 
-.menu {
+.menu, .menuTriggerWidth {
```

Comment:
why'd we add this here?

Reply:
For [this example](https://github.com/adobe/react-spectrum/pull/5465/files/f592d7d5862ae0b8d9215ba9fc6fdcf0d7e2ad71#diff-7ae373998263ffc95c0231c5f6b5b6538b2d3b2ae18056d8c016f5bb631e7cc5R322). I could change the example to be "menu menuTriggerWidth". I planned the Storybook example would go away before merging. I just wanted to make initial testing easier.